manage.py

```diff
@@ -306,3 +307,8 @@ def config(config_key):
         {k: v for k, v in app.config.items() if isinstance(v, basestring)},
         indent=2,
     ))
+
+
+@app.cli.command()
+def initialize_celery_beat_healthcheck():
```

Comment:
Maybe something more obvious, like `celery_beat_healthcheck` or `set_celerybeat_healthy`? Could be used outside of initialization/deployment

Reply:
I liked ```set_celerybeat_healthy```